src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -117,7 +117,7 @@ public async Task<IDelayedException> TryExecuteAsync(IFunctionInstance functionI
                     if (instanceLogEntry != null)
                     {
                         await NotifyCompleteAsync(instanceLogEntry, functionStartedMessage.Arguments, exceptionInfo);
-                        _resultsLogger?.LogFunctionResult(instanceLogEntry);
+                        _resultsLogger?.LogFunctionResult(instanceLogEntry, exceptionInfo?.SourceException.ToString());
```

Comment:
This is the wrong place for this -- the FunctionResult is meant to have just data, no strings. `_functionInstanceLogger.LogFunctionCompletedAsync ()`is where we write out the Completed message in Kusto, etc.

But that should have the exception logged to it -- maybe we're losing the stack somewhere in this? 

```
functionStartedMessage.Failure = new FunctionFailure
{
    Exception = exception,
    ExceptionType = exception.GetType().FullName,
    ExceptionDetails = exception.ToDetails(),
};
```

Reply:
That was where I started my investigation with. Whether exception.ToDetails() was losing the stack while prettifying it somewhere. But turns out that's not the case. I verified the stack trace being present all the way in https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs#L119

I also noticed it doesn't go anything with failure.ExceptionDetails. Maybe pass it as another string parameter there ?